### PR TITLE
Update E2E tests

### PR DESCRIPTION
**Pull request by ezequielc**
## Description
Update E2E tests for pricing page

## Motivation and Context
Checkout flow epic

## How Has This Been Tested?
Tests pass
[stage-1]

## Release Plan:
- As the Submitter, upon requesting review of this pull request, I confirm that the [Release Checklist](https://help.risevision.com/hc/en-us/articles/360031119991) has been completed. 
- As the Reviewer, upon approving the changes in this PR, I confirm I have reviewed and I agree that the [Release Checklist](https://help.risevision.com/hc/en-us/articles/360031119991) has been completed

#### Release Checklist Items Skipped?
If any Release Checklist items were intentionally skipped, please provide which ones and the reasons why

### CODE REVIEW

File: test/e2e/registration/cases/checkout.js [hunk 1]
```diff
@@ -44,22 +44,22 @@
           expect(purchaseFlowPage.getPlanSubscribeLink().isDisplayed()).to.eventually.be.true;
         });
 
-        it("should open plans modal", function() {
+        it("should open plans page", function() {
 
           helper.clickWhenClickable(purchaseFlowPage.getPlanSubscribeLink(), 'Subscribe Button');
 
-          helper.wait(pricingComponentModalPage.getSubscribeButton(), 'Pricing Component Modal');
+          helper.wait(pricingPage.getSubscribeButton(), 'Pricing Component Modal');
```
Comment (alex-deaconu) [inline test/e2e/registration/cases/checkout.js:51]: @ezequielc no need to address in this PR but this still references the `Modal`. Thanks.

File: web/partials/purchase/checkout-plan-picker.html [hunk 2]
```diff
@@ -95,7 +95,7 @@
        
         <div class="button-toolbar flex-row">
           <a href="https://www.risevision.com/pricing" target="_blank" class="btn btn-default btn-block btn-hg font-weight-bold">Send Me A Quote</a>
-          <button type="submit" class="btn btn-primary btn-block btn-hg font-weight-bold" ng-disabled="pricingForm.$invalid">Subscribe</button>
+          <button type="submit" id="subscribeButton" class="btn btn-primary btn-block btn-hg font-weight-bold" ng-disabled="pricingForm.$invalid">Subscribe</button>
```
Comment (alex-deaconu) [inline web/partials/purchase/checkout-plan-picker.html:98]: @ezequielc to confirm, this is the only fix the rest are just wording? Thanks.

**Review by alex-deaconu — APPROVED** [review]
@ezequielc looks good 👍

File: web/partials/purchase/checkout-plan-picker.html (hunk 2 above)
Comment (ezequielc) [reply web/partials/purchase/checkout-plan-picker.html:98]: @alex-deaconu That is correct. Thanks!

File: test/e2e/registration/cases/checkout.js (hunk 1 above)
Comment (ezequielc) [reply test/e2e/registration/cases/checkout.js:51]: @alex-deaconu Sounds good. Thanks!

**Pull request merged**